contracts/interchain-token-service/src/contract/query.rs

```diff
@@ -34,6 +35,54 @@ pub fn all_its_contracts(deps: Deps) -> Result<Binary, Error> {
     to_json_binary(&contract_addresses).change_context(Error::JsonSerialization)
 }
 
+fn convert_chain_filter_option(
+    message_chain_filter: Option<msg::ChainFilter>,
+) -> impl Fn(&state::ChainConfig) -> bool + 'static {
+    move |config| match &message_chain_filter {
+        None => true,
+        Some(filter) => convert_chain_filter(filter)(config),
+    }
+}
```

Comment:
```suggestion
fn convert_chain_filter_option(
    message_chain_filter: Option<msg::ChainFilter>,
) -> impl Fn(&state::ChainConfig) -> bool + 'static {
    let chain_filter = message_chain_filter.unwrap_or_default();

    move |config| convert_chain_filter(&chain_filter)(config)
}
```
And add `#[derive(Default)]` to `ChainFilter`. The idea is that the default filter should be NOT filtering anything.